gravitee-apim-console-webui/src/management/api/proxy/endpoints/components/endpoint-http-config/endpoint-http-config.component.ts

```diff
@@ -225,20 +243,31 @@ export class EndpointHttpConfigComponent implements OnInit, OnDestroy {
       )
       .subscribe((enabled) => {
         if (enabled === true) {
-          httpProxy.get('useSystemProxy').enable();
-          httpProxy.get('host').enable();
-          httpProxy.get('port').enable();
-          httpProxy.get('type').enable();
-          httpProxy.get('username').enable();
-          httpProxy.get('password').enable();
+          httpProxy.get('useSystemProxy').enable({ onlySelf: true });
+          httpProxy.get('host').addValidators(Validators.required);
+          httpProxy.get('host').enable({ onlySelf: true });
+          httpProxy.get('port').addValidators(Validators.required);
+          httpProxy.get('port').enable({ onlySelf: true });
+          httpProxy.get('type').enable({ emitEvent: false });
+          httpProxy.get('username').enable({ emitEvent: false });
+          httpProxy.get('password').enable({ emitEvent: false });
```

Comment:
I don't know if it's a good idea and you can let it like that but it looks like some part of the code could be mutualised between this valueChanges and the one from useSystemProxy no ? 🤷🏼